src/routes/Manage/UserTable/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { ISearchData } from 'types/types'
+import TableRow from './TableRow'
+import cx from 'classnames'
+
+import styles from './userTable.module.scss'
+import React from 'react'
+
+// TODO : props로 { searchData }: { searchData: ISearchData[] } 추가
+
+const DUMMY_USER_LIST: ISearchData[] = [
+  {
+    userNumber: 2,
+    createdAt: '2022-03-02 12:34:56',
+    userStatus: '정상 회원',
+    id: 'minsu12',
+    nickname: '건강최고',
+    birth: '1990년',
+    gender: '남',
+    managerId: null,
+  },
+  {
+    userNumber: 3,
+    createdAt: '2022-02-02 12:34:56',
+    userStatus: '휴면 회원',
+    id: 'jisu22',
+    nickname: '건강왕',
+    birth: '1997년',
+    gender: '여',
+    managerId: null,
+  },
+  {
+    userNumber: 7,
+    createdAt: '2020-01-02 12:34:56',
+    userStatus: '정상 회원',
+    id: 'koko110',
+    nickname: '코코호도',
+    birth: '2000년',
+    gender: '남',
+    managerId: 'manager123',
+  },
+]
+
+const UserTable = () => {
+  const EmptySearchList = React.createElement('span', {}, '일치하는 회원이 없습니다.')
+
+  const CntSearchList = React.createElement(
+    React.Fragment,
+    null,
+    React.createElement('span', null, '전체 총'),
+    React.createElement('span', null, `${DUMMY_USER_LIST.length}`),
+    React.createElement('span', null, '명의 회원이 검색되었습니다.')
+  )
```

Comment:
이거 어디서쓰시는부분인가요 ?
그냥 컴포넌트로 만들어서 불러오셔도 될것같아요

Reply:
이거 하단에 메세지 부분인데 기능없이 
```html
<span>일치하는 회원이 없습니다.</span>

<>
  <span></span>
</>
```

Reply:
컴포넌트로 꼭 안빼도 괜찮지 않을까요 ?? 어떻게 하는게 나을지 잘 모르겠네요 😅

Reply:
```suggestion
  const EmptySearchList = <span>일치하는 회원이 없습니다.</span>

const CntSearchList = (
  <>
    <span>전체 총</span>
    <span>{DUMMY_USER_LIST.length}</span>
    <span>명의 회원이 검색 되었습니다.</span>
  </>
)
```
아하 컴포넌트로 분리하긴 확실히 애매하네요 😂 
createElement 없이 위처럼 하셔두 똑같이 동작 할거에용
createElement는 약간 document.createElement 같은 느낌이라 애매한 느낌이 들었거든요

Reply:
첨에 저렇게 했을때 안됐는데 다시 작성하니까 잘되네요,, 제가 뭘 잘못 작성했었나봐요 ㅋㅋㅋㅋ 이 의견까지 반영해서 기능 정상작동하도록 수정했습니다!